DESCRIPTION

```diff
@@ -15,8 +15,14 @@ RoxygenNote: 7.1.2
 Depends:
   madrat,
   magclass,
-  mrcommons
+  mrcommons,
+  mrremind
```

Comment:
What functions from mrremind are still used here?

Reply:
`calcIO` has to move from mrremind to mrcommons.